pkg/client/goboapi_test.go

```diff
@@ -36,7 +36,7 @@ func TestGoboApi(t *testing.T) {
 		}
```

Comment:
Her ville jeg gjerne sett enhetstester som også sender inn andre varianter av input til vars enn nil til api.RunGraphQl() for å teste at parsingen av dette går som det skal.

Reply:
Fikser